simplecloud-modules/simplecloud-module-placeholders/src/main/kotlin/eu/thesimplecloud/module/placeholders/spigot/BukkitPluginMain.kt

```diff
@@ -0,0 +1,46 @@
+package eu.thesimplecloud.module.placeholders.spigot
+
+import eu.thesimplecloud.module.placeholders.spigot.placeholderapi.PlaceholderHook
+import org.bukkit.plugin.java.JavaPlugin
+
+class BukkitPluginMain : JavaPlugin() {
+
+    var replacePermissionModule = false
+    var replaceChatTabModule = false
```

Comment:
What are those variables used for?

Reply:
Now it's being done by checking if the plugin is null with checking if Bukkit#getPluginManager#getPlugin isn't null, fields are now `val` (final) and aren't having a setter